lib/ezcater_rubocop/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module EzcaterRubocop
-  VERSION = "3.0.0"
+  VERSION = "3.1.0"
```

Comment:
If we do decide that unpinning the gem versions is OK after all, I think it would warrant a major version bump.

Reply:
Sure, makes sense.